### PR TITLE
Changelog for 1.4.0-rc3

### PR DESCRIPTION
**Pull request by lafriks**
As title

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Review by jonasfranz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.4.0-rc3](https://github.com/go-gitea/gitea/releases/tag/v1.4.0-rc3) - 2018-03-16
+* SECURITY
+  * Refactor and simplify to coorectly validate redirect to url (#3674) (#3676)
```
Comment (bugreport0) [inline CHANGELOG.md]: I think we want 'correctly' and 'URL' in caps.

[the remark this answers is not in the document]
Comment (lafriks) [reply CHANGELOG.md]: Thanks, fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.4.0-rc3](https://github.com/go-gitea/gitea/releases/tag/v1.4.0-rc3) - 2018-03-16
+* SECURITY
+  * Refactor and simplify to correctly validate redirect to URL (#3674) (#3676)
+* BUGFIXES
+  * Update markbates/goth libary to fix OAuth2 support (#3661) (#3663)
```
Comment (jonasfranz) [inline CHANGELOG.md]: library instead of libary

**Review by jonasfranz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (lafriks) [reply CHANGELOG.md]: Fixed

**Review by jonasfranz — APPROVED** [review]

**Pull request merged**